tfx/components/example_gen/base_example_gen_executor.py

```diff
@@ -268,7 +262,7 @@ def Do(
       input_dict: Input dict from input key to a list of Artifacts. Depends on
         detailed example gen implementation.
       output_dict: Output dict from output key to a list of Artifacts.
-        - examples: splits of tf examples.
+        - examples: splits of TF records.
```

Comment:
please update the rest of the docstring (e.g., in Do()
--the output is indented to be serialized tf.train.Examples protocol buffer )